src/EditorFeatures/Core.Cocoa/Options/VisualStudioMacOptionsStorage.cs

```diff
@@ -64,14 +64,14 @@ public string GetName(string? language)
         {"dotnet_sort_system_directives_first", new PropertyName("TextEditor.%LANGUAGE%.Specific.PlaceSystemNamespaceFirst")},
         {"dotnet_separate_import_directive_groups", new PropertyName("TextEditor.%LANGUAGE%.Specific.SeparateImportDirectiveGroups")},
         {"QuickInfoOptions_ShowRemarksInQuickInfo", new PropertyName("TextEditor.%LANGUAGE%.Specific.ShowRemarks")},
-        {"FeatureOnOffOptions_AddImportsOnPaste", new PropertyName("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2")},
+        {"dotnet_add_import_on_paste_options_add_imports_on_paste", new PropertyName("TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2")},
```

Comment:
Tag maybe @davidwengier  to review. I am changing the option name of these options. Will I break anything if things also renamed in this table?

Reply:
There are a couple of spots where VS Mac references `FeatureOnOffOptions` that will need to be updated, but changing this list shouldn't cause problems. Not updating it would have though :)

FYI @sandyarmstrong who knows more about the VS Mac side of this

Reply:
Yeah this should be fine, the keys can change here but the values need to stay the same.

Reply:
@sandyarmstrong  I am also moving the options to a different class, would this break VS mac?

Reply:
No, just means when I do the next insertions I update the `FeatureOnOffOptions.Blah` code to `WhatzitOptions.Blah`. Shouldn't be a problem (unless the new class is in a project that doesn't have IVT to VSMac, in which case i'd have to update that first)